lib/happy_hood/slack/client.rb

```diff
@@ -4,18 +4,12 @@ class Client
       DefaultSlackChannel = "#happy-hood".freeze
       DefaultIconEmoji = ":house_buildings:".freeze
 
-      def self.send_daily_price_summary
-        message = daily_message
+      def self.send_summary(start_date:, end_date:, error_text: nil)
```

Comment:
This is the big change here. If you see the changes commit-by-commit, you'll see that this method was first added. Afterward, each `send_daily/monthly` method was updated to use the same message building method (tests passed).